### DISCUSSION TITLE
Blogpost 2

### DISCUSSION THREAD

**Discussion by Turbo3A** [LeonRother/SE_Platzverwaltung#3] [General]
In dieser Woche haben wir uns ausschließlich mit der SRS für unser Projekt beschäftigt. Dazu haben wir gemeinsam Anforderungen & Spezifikation unserer Anwendung definiert. Teil davon war das Ausarbeiten von use cases, die dann von jeweils einem Gruppenmitglied genauer ausgearbeitet wurden.
Diesem Blogeintrag wird das fertige Formular mit der SRS angefügt.

**Comment by stefanschfr** [comment]
Hallo,
Euer SRS-Dokument macht einen durchdachten und strukturierten Eindruck. Besonders die klaren Anforderungen und der Fokus auf Benutzerfreundlichkeit und Barrierefreiheit gefallen mir gut. Die umfassende Beschreibung der Funktionen gibt einen guten Überblick über das geplante System und zeigt, dass ihr die verschiedenen Benutzerrollen und deren Bedürfnisse gut berücksichtigt habt.

Ein kleiner Hinweis: Im Blogpost wird erwähnt, dass das fertige Formular mit der SRS angefügt ist, allerdings ist das SRS dort nicht verlinkt. Außerdem verwendet ihr aktuell drei unterschiedliche Dateiformate. Es wäre hilfreich, sich auf ein einheitliches Format zu einigen, um die Struktur konsistent und übersichtlich zu halten.

Insgesamt ist das Dokument sehr gelungen – tolle Arbeit!

Viele Grüße

Sebastian Albert, Oskar, Luca, Philip und Stefan

**Comment by Hpixpoke** [comment]
Das SRS liegt jetzt als PDF mit Mockups bereit. Ist zwar leider immernoch wip, but we're getting there...
